Sources/Testing/testUtils.js

```diff
@@ -5,14 +5,15 @@ let REMOVE_DOM_ELEMENTS = true;
 function compareImages(image, baselines, testName, tapeContext, threshold = 0.5, nextCallback = null) {
   const nbBaselines = baselines.length;
   let minDelta = 100;
+  let minDiff = new Image();
```

Comment:
minDiff is a string that represent an encoded image, therefore, you can not initialize it as an image. ;-)

`let minDiff = '';`